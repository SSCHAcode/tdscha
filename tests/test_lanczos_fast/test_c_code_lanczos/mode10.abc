2.1958244463998568e-07	4.0276024921473846e-11	1.5733889154384645e-02
3.6468697303092626e-06	8.3559039317101201e-07	9.0498801175471667e-07
3.0534049459196905e-06	5.0895292523203105e-07	2.4681927813924252e-06
5.9420421098876791e-07	1.3307855172936484e-06	2.5065197880233540e-07
3.2293720829509187e-06	7.2428970213566092e-07	6.5974434410687537e-07
