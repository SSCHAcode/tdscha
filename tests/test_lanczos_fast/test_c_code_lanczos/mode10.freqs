3.602992689378278594e-04
3.602992689378278594e-04
3.602992689378278594e-04
3.602992689378278594e-04
3.602992689378278594e-04
3.602992689378278594e-04
4.590666396271931692e-04
4.590666396271931692e-04
4.590666396271931692e-04
4.685962490673455756e-04
4.685962490673455756e-04
4.685962490673456298e-04
6.589746567961170557e-04
6.589746567961170557e-04
6.589746567961170557e-04
6.589746567961170557e-04
6.589746567961170557e-04
6.589746567961170557e-04
6.915815435516317958e-04
6.915815435516317958e-04
6.915815435516317958e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
7.415861385477882841e-04
8.951215637471479462e-04
8.951215637471479462e-04
8.951215637471479462e-04
8.951215637471479462e-04
8.951215637471480547e-04
8.951215637471480547e-04
8.951215637471480547e-04
8.951215637471480547e-04
9.772492677090878835e-04
9.772492677090878835e-04
9.772492677090878835e-04
9.772492677090878835e-04
1.106016120766416429e-03
1.106016120766416429e-03
1.106016120766416429e-03
1.106016120766416429e-03
