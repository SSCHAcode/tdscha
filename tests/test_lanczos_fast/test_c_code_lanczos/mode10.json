{"T": 250, "n_steps": 5, "ignore_v2": false, "ignore_v3": false, "ignore_v4": false, "use_wigner": false, "run_sym": false, "data": {"n_configs": 10, "n_modes": 45, "n_syms": 384, "n_blocks": 9, "perturbation_modulus": 1, "reverse": false, "shift": 0}}