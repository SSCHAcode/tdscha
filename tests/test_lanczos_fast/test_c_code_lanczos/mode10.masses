1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.163002852960780001e+05
1.081975459835219954e+05
1.081975459835219954e+05
1.081975459835219954e+05
