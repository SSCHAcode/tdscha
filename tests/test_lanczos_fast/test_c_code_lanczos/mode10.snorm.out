1.0000000000000000e+00
5.3914777797987999e-02
6.4329928008835877e-06
1.7142913899868234e-05
8.6067986924179996e-07
1.6493409833640450e-06
